website/content/en/docs/reference/cloudformation.md

```diff
@@ -211,7 +211,7 @@ The AllowScopedResourceTagging Sid allows EC2 [CreateTags](https://docs.aws.amaz
   "Action": "ec2:CreateTags",
   "Condition": {
     "StringEquals": {
-      "aws:ResourceTag/karpenter.sh/cluster/${ClusterName}": "owned"
```

Comment:
Good catch 🎉 . Do you mind checking if there are other places where this happened alongside validating that every version of the docs gets this fix (right now this change is just on the `/docs` path)

Reply:
I added commit with fix the incorrect tag in another version of the document :) thx for comments